### DISCUSSION TITLE
Does Sumatra PDF support reading PDF/A-3 and/or PDF/A-4 files?

### DISCUSSION THREAD

**Discussion by normag** [sumatrapdfreader/sumatrapdf#4589] [General]
any problems viewing these type of files?

**Comment by GitHubRulesOK** [comment]
Why would there be any difference between PDFv1 with XML or v2  viewing or any other variation
is the use of extra exotic features thus the PDF print format part is was always viewable
but for example Adobe Proprietary XFA would be considered out of ISO PDF viewer standards.

PDF/Adobe/AssociationType3 (ISO 32000-1 with support for embedded files) and /Adobe/AssociationType4 (Use of ISO 32000-2) are super sets thus have things like needing enhanced imbedded XML viewers etc which are not PDF basic functionality

PDF/A was initially intended to ENSURE baseline /ARCHIVAL but that is now drifting ever upwards with more commercial reasons !
PDF/A1
- Audio and video content are forbidden.
- JavaScript and executable file launches are forbidden.
- Encryption is forbidden
- External content references are forbidden. etc 

so what will PDF/A6 be will it be exploitable ShockWaves again?

PDF/A3 differs from PDF/A-2 in only one regard: it allows embedding of arbitrary file formats (such as XML, CSV, CAD, word-processing documents, spreadsheet documents, and others) into PDF/A 

PDF/A4 is ISO 2 and SumatraPDF can read those it just will not show any type /A3 contents as exploitables in 3.6. (for that go back to earlier versions like 3.5.2 )
![image](https://github.com/user-attachments/assets/5fae6cd9-8508-4986-9a30-36c04c4d84e6)
![image](https://github.com/user-attachments/assets/fd47584b-9c59-494e-b46d-387686f5f05f)
